03-aad.md

```diff
@@ -1,72 +1,85 @@
+   ```
+
+1. Playing the role as the Contoso Bicycle Azure AD team, login into the tenant where Kubernetes Cluster API authorization will be associated with.
+
+   ```bash
+   az login -t <Replace-With-ClusterApi-AzureAD-TenantId> --allow-no-subscriptions
+   TENANTID_K8SRBAC=$(az account show --query tenantId -o tsv)
+   ```
+
+1. Create/identify the Azure AD security group that is going to map to the [Kubernetes Cluster Admin](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles) role `cluster-admin`.
+
+   If you already have a security group that is appropriate for your cluster's admin service accounts, use that group and skip this step. If using your own group or your Azure AD administrator created one for you to use; you will need to update the group name and ID throughout the reference implementation.
+
+   ```bash
+   export AADOBJECTID_GROUP_CLUSTERADMIN=$(az ad group create --display-name 'cluster-admins-bu0001a000800' --mail-nickname 'cluster-admins-bu0001a000800' --description "Principals in this group are cluster admins in the bu0001a000800 cluster." --query objectId -o tsv)
+   ```
+
+   This Azure AD group object ID will be used later while creating the cluster. This way, once the cluster gets deployed the new group will get the proper Cluster Role bindings in Kubernetes.
+
+1. Create a "break-glass" cluster administrator user for your AKS cluster.
+
+   > :book: The organization knows the value of having a break-glass admin user for their critical infrastructure. The app team requests a cluster admin user and Azure AD Admin team proceeds with the creation of the user in Azure AD.
+
+   ```bash
+   export TENANTDOMAIN_K8SRBAC=$(az ad signed-in-user show --query 'userPrincipalName' -o tsv | cut -d '@' -f 2 | sed 's/\"//')
+   export AADOBJECTNAME_USER_CLUSTERADMIN=bu0001a000800-admin
+   export AADOBJECTID_USER_CLUSTERADMIN=$(az ad user create --display-name=${AADOBJECTNAME_USER_CLUSTERADMIN} --user-principal-name ${AADOBJECTNAME_USER_CLUSTERADMIN}@${TENANTDOMAIN_K8SRBAC} --force-change-password-next-login --password ChangeMebu0001a0008AdminChangeMe --query objectId -o tsv)
+   ```
+
+1. Add the cluster admin user(s) to the cluster admin security group.
+
+   > :book: The recently created break-glass admin user is added to the Kubernetes Cluster Admin group from Azure AD. After this step the Azure AD Admin team will have finished the app team's request.
+
+   ```bash
+   az ad group member add -g $AADOBJECTID_GROUP_CLUSTERADMIN --member-id $AADOBJECTID_USER_CLUSTERADMIN
+   ```
+
+1. Create/identify the Azure AD security group that is going to be a namespace reader.
+
+   ```bash
+   export AADOBJECTID_GROUP_A0005_READER=$(az ad group create --display-name 'cluster-ns-a0005-readers-bu0001a000800' --mail-nickname 'cluster-ns-a0005-readers-bu0001a000800' --description "Principals in this group are readers of namespace a0005 in the bu0001a000800 cluster." --query objectId -o tsv)

```

Comment:
Why is this called "A0005" when the namespace is `a0008`?  Looks like you have it as "0005" in a couple of places.  Can you update them all?  Thanks.

Reply:
Renamed !